docs/source/commands.rst

```diff
@@ -161,3 +162,48 @@ command will notify the user by sending everything either to Telegram or RocketC
      - Logging level, one of ``debug``, ``info``, ``warning``, ``error``, ``critical``.
        Default: ``error``.
 
+
+.. _command-check-initiate-l1-invalidations:
+
+``beamer check initiate-l1-invalidations``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``beamer check initiate-l1-invalidations read [OPTIONS] PROOF_SOURCE PROOF_TARGET...``
+
+The command initiates L1 invalidations from ``PROOF_SOURCE`` to each
+``PROOF_TARGET``. ``PROOF_SOURCE`` and ``PROOF_TARGET`` are chain IDs. By
```

Comment:
Could you extend with a sentence that the last N arguments (but at least 1) are PROOF_TARGETS. At least it is not clearly understandable that you can add many proof targets.

Reply:
Eh, this is just using the `X...` syntax with triple dots to describe arbitrary number of args (at least one), which is pretty standard, see e.g. `cat` manpage:
```
NAME
       cat - concatenate files and print on the standard output

SYNOPSIS
       cat [OPTION]... [FILE]...
```

But I will add some wording to make it more explicit.

Reply:
Yes, I see it in the command, of course. But I was specifically referring to the wording.